bin/deploy-backer-lender

```diff
@@ -11,33 +11,28 @@ export SOLC_FLAGS="--optimize"
 
 test -z "$SKIP_BUILD" && dapp build
 
-# Set god address to 0 if not defined
-test -z "$GOD_ADDR" && GOD_ADDR="0x00000000000000000000"
+if [[ -z "$BACKER_ADDR" ]]; then
+    BACKER_ADDR="0x00000000000000000000000000000000000000000000"
+fi
 
-LENDER_FAB=$(dapp create BackerFab $BACKER_ETH_FROM)
+LENDER_FAB=$(dapp create BackerFab $BACKER_ADDR)
 
 $(seth send $DEPLOYER 'deployLender(address, address)' $CURRENCY $LENDER_FAB)
 LENDER=0x$(seth call $DEPLOYER 'lender()(address)')
 $(seth send $DESK 'file(bytes32,address)' "$(seth --to-bytes32 "$(seth --from-ascii "lender")")" $LENDER)
 $(seth send $LENDER 'rely(address)' $DESK)
 
+if [[ -n "$BACKER_ETH_FROM" ]]; then
+    seth send $CURRENCY 'approve(address,uint)' $LENDER 100000 --keystore=$BACKER_ETH_KEYSTORE --password=$BACKER_ETH_PASSWORD --from=$BACKER_ETH_FROM
```

Comment:
```suggestion
    seth send $CURRENCY 'approve(address,uint)' $LENDER ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff --keystore=$BACKER_ETH_KEYSTORE --password=$BACKER_ETH_PASSWORD --from=$BACKER_ETH_FROM
```

Reply:
I would move it into an additional script. 
`--keystore=$BACKER_ETH_KEYSTORE --password=$BACKER_ETH_PASSWORD ` is not always the case. If we are using a ledger we only need `$BACKER_ETH_FROM`